### PR TITLE
Python: Converting suggestions List to string t

### PR DESCRIPTION
**Pull request by rajrajput79**
Python: Converting suggestions List to string to display output incorrect format

### CODE REVIEW

**Comment by rajrajput79** [comment]
Word should be display in proper format.
Each word on next line

**Comment by sujankandela** [comment]
tried and tested it working if possible try to give space between the words

**Comment by HalimaMalik** [comment]
Try and tested

File: src/team03.py [hunk 1]
```diff
@@ -73,15 +73,34 @@ def spellCheck(text,languagetowrite): # Main function
...
+            print("e")
+            print(e)
+            x= ','.join([str(element) for element in e])
+            print("x")
+            print(x)
+            word=word + x + "\n"
+            suggestions.append(word)
+            # appending suggestion if there are more than a single word 
+            print(suggestions)
+            print("suggestions")
```
Comment (NagaGanesh1315) [inline src/team03.py:103]: Tested this "converting word into string" @rajrajput79 . Good job!

**Pull request merged**